### DISCUSSION TITLE
Multilple attempts to install Masa 7.4

### DISCUSSION THREAD

**Discussion by joseph-krogman** [MasaCMS/MasaCMS#189] [Q&A] [answered]
Hello Everyone

I have tried several times over the past 8+ months to install Masa. I've tried it on 3 different computers with different versions of Masa and I can't seem to get a successful install (I've posted here previously).

Just today, I tried 7.4.1 on a VM Windows 2019 server with MSSQL 2019 and CF14 (2021). I follow all the steps but I eventually give up. Todays error is:

500 Error
The CFML compiler encountered an unexpected java.lang.NullPointerException exception.

Type:Template
The reason for this was: '' Occurred at:<pre>java.lang.NullPointerException at coldfusion.compiler.ASTvariableDefinition.setVariableName(ASTvariableDefinition.java:45) at coldfusion.compiler.cfml40.localVariableDefinition(cfml40.java:6467) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:2576) at coldfusion.compiler.cfml40.cfscriptBlock(cfml40.java:4046) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:2727) at coldfusion.compiler.cfml40.ComplexStatement(cfml40.java:2895) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:2713) at coldfusion.compiler.cfml40.consumeFunctionBody(cfml40.java:5210) at coldfusion.compiler.cfml40.functionDefinition(cfml40.java:5145) at coldfusion.compiler.cfml40.cfscriptStatement(cfml40.java:2732) at coldfusion.compiler.cfml40.cfscript(cfml40.java:2436) at coldfusion.compiler.cfml40.processCfmlTags(cfml40.java:7473) at coldfusion.compiler.cfml40.cfml(cfml40.java:7519) at coldfusion.compiler.cfml40.start(cfml40.java:7988) at coldfusion.compiler.NeoTranslator.parsePag...</pre>
coldfusion.compiler.ParseException$CompilerExceptionWrapper: The CFML compiler encountered an unexpected java.lang.NullPointerException exception.

I'd be very appreciative of any assistance or advice anyone can offer. I really want to use and support this!

JK

**Comment by no3rdw** [comment, marked as answer]
I can't tell for sure if this is the same issue since your error message is cutoff, but I've been getting a Null Pointer Exception when I install a brand new instance of Masa too, and traced it (in my case) to a specific CFC used to display category navigation.  [See this post for details](https://github.com/MasaCMS/MasaCMS/issues/178) and let me know if deleting this file helps.

**Reply by joseph-krogman** [reply]
no3rdw - Thank you SOOO much for you help! This solved the issue. This is the first time I've seen Masa in action since Mura moved back to private license. I'm very appreciative for your help. JK
